ingestion/src/metadata/ingestion/ometa/mixins/table_mixin.py

```diff
@@ -85,6 +86,22 @@ def ingest_table_data_model(self, table: Table, data_model: DataModel) -> Table:
         )
         return Table(**resp)
 
+    def ingest_table_queries_data(
+        self, table: Table, table_queries: List[SqlQuery]
+    ) -> None:
+        """
+        PUT table queries for a table
+
+        :param table: Table Entity to update
+        :param table_queries: SqlQuery to add
+        """
+        for query in table_queries:
+            resp = self.client.put(
+                f"{self.get_suffix(Table)}/{table.id.__root__}/tableQuery",
+                data=query.json(),
+            )
+        return None
```

Comment:
Code Smell here..
can you remove the unused variable here..
instead of returning None, we can return Table Object here.